drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -776,7 +776,7 @@ static int rf2xx_init(struct device *dev)
 			dev, NULL, NULL,
 			K_PRIO_COOP(2), 0, K_NO_WAIT);
 
-	sprintf(thread_name, "802.15.4 main [%d]", conf->inst);
+	sprintf(thread_name, "rf2xx_rx [%d]", conf->inst);
```

Comment:
Could you use `snprintk()` here instead, it uses less resources.

Reply:
@nandojve FYI.

Reply:
I agreed with Jukkar, it will be better switch to a safer version of sprintf like snprintk.
I agreed that 802.15.4 could not be the better prefix and should be switched to rf2xx.

At "802.15.4 main" thread all interrupts are processed, RX code and it releases TX. The rf2xx is a multi instance driver and you are trying to normalize it with an "rx" suffix. You should keep the instance number to identify the thread. The 802,15,4 drivers have been improved on 2.3 and rf2xx not follow all changes yet. In future we would process even more interrupts and control power management inside this thread.

The `rf2xx_process_trx_end` method is processed at end of thread and selects between receive code or release tx.

I can think some names like: 'rf2xx_main [n]', 'rf2xx_fsm [n]' etc but 'rf2xx_rx [n]' not seems to be the better name.
Is it make sense? Can we have others alternatives?

Reply:
how about 'rf2xx_trx [n]' since the thread is also called trx_thread.

Reply:
Sure, do it!

Reply:
done.